### PR TITLE
Closes AgileVentures/MetPlus_tracker#300

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: spec/models/concerns/task_manager/business_logic_spec.rb [hunk 1]
```diff
@@ -73,4 +73,24 @@ class TaskTester
...
   end
+
+  describe 'Review job application' do
+    let(:job_seeker)     { FactoryGirl.create(:job_seeker) }
+    let(:company)        { FactoryGirl.create(:company) }
+    let(:company_admin)  { FactoryGirl.create(:company_admin, company: company) }
+    let(:job)            { FactoryGirl.create(:job, company: company,
+                                              company_person: company_admin) }
+
+    # before :each do
```
Comment (joaopapereira) [inline spec/models/concerns/task_manager/business_logic_spec.rb]: Why is this commented?

File: app/models/job.rb [hunk 2]
```diff
@@ -37,4 +37,7 @@ def apply job_seeker
     save!
   end
 
+  def last_application_by_job_seeker(job_seeker)
+    job_applications.where(job_seeker: job_seeker).last
```
Comment (joaopapereira) [inline app/models/job.rb]: This should have one order by creation_date

**Comment by joaopapereira** [comment]
Looks good, please review my comments

File: spec/models/concerns/task_manager/business_logic_spec.rb (hunk 1 above)
Comment (patmbolger) [inline spec/models/concerns/task_manager/business_logic_spec.rb]: That code was not needed.  I will remove the comment.

**Comment by patmbolger** [comment]
Adjusted per comments.  Changes made to:

- app/models/job.rb
- spec/models/concerns/task_manager/business_logic_spec.rb
- spec/models/job_spec.rb

**Comment by joaopapereira** [comment]
Looks good going to merge

**Pull request merged**